etl/etl/utils.py

```diff
@@ -121,3 +121,27 @@ def commaize_number(number):
         return retn
     except Exception as e:
         return number
+
+
+def parse_pare(pare_option):
+    """Parses the PARE_OPTION and returns a tuple of
+    (PARE_FACTOR, KEYS_TO_INCLUDE) where PARE_FACTOR is an int
+    by which proposals should be reduced, and KEYS_TO_INCLUDE
+    is a list of proposals to include by key.  Only one of the
+    two will ever be returned."""
+
+    pare_factor = None
+    keys_to_include = None
+    if pare_option is None:
+        pass
+    elif pare_option.startswith("@"):
+        with open(pare_option[1:]) as pare_file:
+            keys_to_include = [l.strip().split(' ')[0] for l in pare_file.readlines() if l.strip()]
+    elif pare_option.startswith("+"):
+        keys_to_include = pare_option[1:].split(",")
+    else:
+        try:
+            pare_factor = int(pare_option)
+        except Exception:
```

Comment:
Is `except Exception` different from just `except`?  (Maybe my Python-fu is out of date?)

Reply:
I didn't know about that, cool!